e2e-tests/portal/cypress/integration/pages/facilitiesDashboard.js

```diff
@@ -1,5 +1,5 @@
 const page = {
-  visit: () => cy.visit('/dashboard/facilities'),
+  visit: () => cy.visit('/dashboard/facilities/gef'),
```

Comment:
Is this GEF specific test, if so should the file be renamed with GEF?

Reply:
good point - this is page selectors file rather than an actual test. Could move this around but very soon it will be generic "facilities", feels pointless to move it to be GEF specific or BSS specific right now as it's only going to change